common/src/lib.rs

```diff
@@ -1,2 +1,4 @@
 pub mod env_vars;
 pub mod paths;
+
+pub const DEFAULT_PASSPHRASE: &str = "convenient and insecure keystore";
```

Comment:
Please add a TODO comment here that this is going away as soon as we can have a single passphrase for keystore bundle.